pom.xml

```diff
@@ -1235,7 +1256,6 @@
                             <excludes>
                                 <exclude>org.apache.shardingsphere.agent.core.**</exclude>
                             </excludes>
-                            <argLine>--add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/java.lang.reflect=ALL-UNNAMED</argLine>
```

Comment:
what will happen if execute on JDK 8 or 17 without this argument ?

Reply:
- It is impossible for the Maven Profile of `generateStandardMetadata` to run on JDK8.  This Maven Profile is restricted to run on JDK17-JDK21 through `maven-compiler-plugin`.  

- In addition, the Maven Profile of `jdk11+` also defines the same.